runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonOverlay.java

```diff
@@ -129,9 +130,13 @@ private void drawDoubleHitSpots(Graphics2D graphics, LocalPoint startTile, Color
 				int yPos = startTile.getY() - (y * LOCAL_TILE_SIZE);
 
 				LocalPoint marker = new LocalPoint(xPos, yPos);
+
+				WorldArea worldArea = new WorldArea(plugin.getCannonWorldPoint(), 1, 1);
+				boolean hasLineOfSight = worldArea.hasLineOfSightTo(client, WorldPoint.fromLocal(client, marker));
```

Comment:
I don't understand what this code is for. Does this fix some bug that exists in the cannon plugin now? If so, that should be a separate commit, if not a separate PR entirely.

Reply:
Fixing this is fine, but it's unrelated to tracking cannon from vars; please isolate this change to its own commit.